recipes/trio/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "trio" %}
+{% set version = "0.3.0" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "d54fcfb12305bbdaa11843efe22c130acdbc72fa7dafb9c5d777051de3fbd72e" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.tar.gz'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  skip: True  # [py27]
+  number: 0
+  script: python setup.py install  --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - attrs
+    - sortedcontainers
+    - async_generator >=1.6
+    - idna
+    - cffi  # [win]
+
+test:
+  imports:
+    - trio
+    - trio._core
+    - trio._core.tests
+    - trio._core.tests.test_multierror_scripts
+    - trio.testing
+    - trio.tests
+
+about:
+  home: https://github.com/python-trio/trio
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: An async/await-native I/O library for humans and snake people
+  description: ".async/await-native I/O library for Python. Like all async libraries,\n
```

Comment:
Kind of big and with some escape characters that I don't think are needed.

Reply:
Done